### DISCUSSION TITLE
Video quality

### DISCUSSION THREAD

**Discussion by Matias-argentum** [pystardust/ytfzf#470] [Q&A]
Hey guys! Greetings from Argentina! I've got s question: Is it posible to lower the video quality??? What changes should I make to to the code in order to achieve that? Thanks!

**Comment by kkatz151** [comment]
Im not sure if this is related you your discussion, but i have been trying to do the exact same thing.  After a new system and reinstalling ytfzf and my old configs from a month ago, nothing i do to video_pref="" in conf.sh does anything, nor does -f show me any format options. i just get back 0/0 when its searching for video formats.  HOWEVER:  adding ```video_pref="worst"``` to line 1329 changes my format at last.  Im sure this is not ideal, but after spending a good hr on this, i am tired and this i settle for till I can come back and spend more time.  Maybe its the conf.sh, so i renamed it, nope.
![ytfzf](https://user-images.githubusercontent.com/64765505/150953376-fbd36af9-0af0-4728-8301-52e538fa082b.png)

**Reply by kkatz151** [reply]
Disreguard,   https://github.com/pystardust/ytfzf/issues/465 fixes this for me too

**Comment by Euro20179** [comment]
> Hey guys! Greetings from Argentina! I've got s question: Is it posible to lower the video quality??? What changes should I make to to the code in order to achieve that? Thanks!

`ytfzf --video-pref=worst`?

And make sure you're on version `2.x`
